packages/api/cms-api/src/index.ts

```diff
@@ -249,8 +250,7 @@ export { SentryModule } from "./sentry/sentry.module";
 export { AzureAiTranslatorModule } from "./translation/azure-ai-translator.module";
 export { AbstractAccessControlService } from "./user-permissions/access-control.service";
 export { AffectedEntity, AffectedEntityMeta, AffectedEntityOptions } from "./user-permissions/decorators/affected-entity.decorator";
-export { RequiredPermission } from "./user-permissions/decorators/required-permission.decorator";
-export { DisablePermissionCheck } from "./user-permissions/decorators/required-permission.decorator";
+export { DisablePermissionCheck, DisablePermissionCheckType, RequiredPermission } from "./user-permissions/decorators/required-permission.decorator";
```

Comment:
Do we need to add DisablePermissionCheckType to the public API? I can't think of a use case where it would be used in application code.

Reply:
Seems unlikely, I removed it 518014253efa7e6ff1526b5ed8cb5fc1171fe6ca